### PR TITLE
CEDS-1625 - fix back link for arrivals summary page

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: test/views/DepartureSummaryViewSpec.scala [hunk 1]
```diff
@@ -72,6 +72,16 @@ class DepartureSummaryViewSpec extends UnitViewSpec with Stubs with Injector {
       departureSummaryViewOut.getElementById("title").text() mustBe messages("summary.departure.title")
     }
 
+    "have correct back link for depart out of uk" in {
+
+      departureSummaryViewOut.getElementById("link-back").attr("href") must endWith("/transport")
```
Comment (PatrykRudnicki) [inline test/views/DepartureSummaryViewSpec.scala]: Small comment, for url you can use `routes.controllers.TransportController.displayPage().url`

Comment (PatrykRudnicki) [reply test/views/DepartureSummaryViewSpec.scala]: so in that case we can use mustBe not must endWith, but actual approach is correct too

Comment (SteveSugden) [reply test/views/DepartureSummaryViewSpec.scala]: Yes.  That would be better.    I copied a similar test.

File: test/views/ArrivalReferenceViewSpec.scala [hunk 2]
```diff
@@ -70,7 +71,7 @@ class ArrivalReferenceViewSpec extends UnitViewSpec with Stubs with Injector {
       val backButton = createView().getElementById("link-back")
 
       backButton.text() mustBe messages("site.back")
-      backButton.attr("href") must endWith("/consignment-references")
+      backButton.attr("href") mustBe routes.ConsignmentReferencesController.displayPage().url
```
Comment (bambuchaAdm) [inline test/views/ArrivalReferenceViewSpec.scala]: haveHref or something i coś takiego

**Pull request merged**